foodhub/accounts/tasks.py

```diff
@@ -0,0 +1,30 @@
+from accounts.serializers import OrderSerializer
+from accounts.models import Merchant, Store, Order
+from celery import shared_task
+from django.db.utils import OperationalError
+import structlog
+import requests
+
+log = structlog.get_logger()
+
+
+@shared_task(name="create_order", autoretry_for=(OperationalError,), retry_kwargs={
```

Comment:
why adv it served to have it as an async task

Reply:
As per requirement i have updated my tasks.py file